recipes-core/everest/files/0001-Make-correct-phase-count-available-in-API.patch

```diff
@@ -0,0 +1,68 @@
+From 479d80307becae49abb6d7ed0cadd6aaac60c9a4 Mon Sep 17 00:00:00 2001
+From: Martin Lukas <martin.lukas@chargebyte.com>
+Date: Mon, 23 Sep 2024 06:58:32 +0200
+Subject: [PATCH] The available phase count is part of the type
+ "evse_manager::limits". This type is published via mqtt and is used by the
+ API module to interact with external software stacks like Nymea. Per default
+ the available phase count is set to value 1, during the init of the
+ evse_manager interface. The EvseManager determines the count of active phases
+ with help of the min and max phase count values as part of the type
+ "evse_board_support::HardwareCapabilities". This is published by other
+ modules like CbTarragonDriver. But this value is never published via "limits"
+ topic through API module. This leads to wrong values in the Nymea App.
+ Although 3 phases are connected to wallbox, only 1 phase is used for energy
+ management. This commit should fix this.
+
+---
+ modules/EvseManager/EvseManager.cpp           |  2 ++
+ modules/EvseManager/evse/evse_managerImpl.cpp | 14 +++++++-------
+ 2 files changed, 9 insertions(+), 7 deletions(-)
+
+diff --git a/modules/EvseManager/EvseManager.cpp b/modules/EvseManager/EvseManager.cpp
+index 6d64a4a..22e69e6 100644
+--- a/modules/EvseManager/EvseManager.cpp
++++ b/modules/EvseManager/EvseManager.cpp
+@@ -139,6 +139,8 @@ void EvseManager::init() {
+             ac_nr_phases_active = c.min_phase_count_import;
+         }
+ 
++        signalNrOfPhasesAvailable(ac_nr_phases_active);
```

Comment:
I guess this should be done in the ready() method, otherwise it is racy, too? Isn't it?

Reply:
Nevermind, should work since it is in the callback, not in the init() method. I was missleaded by the patch view of Github.